app/src/main/java/io/gnosis/safe/utils/StringUtils.kt

```diff
@@ -24,6 +29,7 @@ import pm.gnosis.utils.asEthereumAddress
 import timber.log.Timber
 import java.util.regex.Pattern
 
+// Is this used anywhere except in the test?
```

Comment:
can you check it?

Reply:
I could not find a place. Since we're not using reflections to call methods, I guess it can be removed.